examples/list_interfaces/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Ullaakut/nmap/v2"
+	"log"
+)
+
+func main() {
+	scanner, err := nmap.NewScanner()
+	if err != nil {
+		log.Fatalf("unable to create nmap scanner: %v", err)
+	}
+
+	interfaceList, err := scanner.GetInterfaceList()
+
+	bytes, err := json.MarshalIndent(interfaceList, "", "   ")
+	if err != nil {
+		log.Fatalf("unable to marshall: %v", err)
+	}
```

Comment:
```suggestion
	bytes, err := json.MarshalIndent(interfaceList, "", "\t")
	if err != nil {
		log.Fatalf("unable to marshal: %v", err)
	}
```

WDYT?

Reply:
I personally prefer spaces instead of a single tab, but it is also good for me.

Reply:
In that case I'd also be fine with 2 or 4 spaces. I suggested a tabulation because 3 spaces is uncommon, and the \t is pretty explicit.